src/main/java/org/cbioportal/cdd/service/internal/ClinicalAttributeMetadataCache.java

```diff
@@ -134,4 +135,24 @@ public void resetCache(boolean force) {
         overridesCache = latestOverridesCache;
         logger.info("resetCache(): refilled overrides cache with " + latestOverrides.size() + " overrides");
     }
+
+    private void fillOverrideAttributeWithDefaultValues(ClinicalAttributeMetadata overrideClinicalAttribute, ClinicalAttributeMetadata defaultClinicalAttribute) {
+        logger.debug("fillOverrideAttributeWithDefaultValues()");
+        if (overrideClinicalAttribute.getDisplayName() == null) {
```

Comment:
I think this is reasonable ... to consider empty string as equivalent to null. This will prevent us from overriding any metadata attribute to become an empty string, but the only one which that would possibly make sense for is Description. And we could use "NA" or something similar if we really need to hide the default description.
Essentially this means : consider an OverrideValue record which has an empty string for the value to be invalid and ignore it. Maybe we should be logging these cases, so that we can fix them or clean them up --- so for that purpose, maybe we should distinguish between null (meaning no OverrideValue record exists) and empty string (a case where the value field in the record was not filled in with anything usable.